src/common/metrics/hospitalizations.tsx

```diff
@@ -138,54 +136,39 @@ function renderStatus(projections: Projections): React.ReactElement {
+  const patientBreakdown = includeBreakdown
+    ? `${nonCovidICUPatients} are filled by non-COVID patients and ${covidICUPatients} are filled by COVID patients.`
+    : '';
+
   return (
     <Fragment>
-      {locationName} has about {totalICUBeds} ICU beds. Based on best available
-      data, {textWeEstimateThatNonCovidPatients} are currently occupied by
-      non-COVID patients. Of the {remainingICUBeds} ICU beds remaining,{' '}
-      {textWeEstimate} {covidICUPatients} are needed by COVID cases, or{' '}
-      {icuHeadroom} of available beds. {textLevel}.
+      {locationName} has reported having {totalICUBeds} staffed adult ICU beds.{' '}
+      {patientBreakdown} Overall, {totalICUPatients} out of {totalICUBeds} (
+      {icuHeadroom}) are filled. {textLevel}.
     </Fragment>
   );
 }
 
 function renderDisclaimer(projections: Projections): React.ReactElement {
-  // TODO(michael): Use regionType. But that's causing `yarn
-  // generate-index-pages` to fail due to circular dependency (I think).
-  if (projections.fips.length === 2) {
-    return (
-      <Fragment>
-        Raw data directly sourced from the{' '}
-        <ExternalLink href="https://healthdata.gov/dataset/covid-19-reported-patient-impact-and-hospital-capacity-state-timeseries">
-          Department of Health and Human Services (HHS)
-        </ExternalLink>
-        . Learn more about our{' '}
-        <ExternalLink href="https://covidactnow.org/covid-risk-levels-metrics#icu-headroom-used">
-          ICU Headroom methodology
-        </ExternalLink>
-        .
-      </Fragment>
-    );
-  } else {
-    return (
-      <Fragment>
-        <ExternalLink href="https://preventepidemics.org/wp-content/uploads/2020/04/COV020_WhenHowTightenFaucet_v3.pdf">
-          Resolve to Save Lives
-        </ExternalLink>
-        , a pandemic think tank, recommends that hospitals maintain enough ICU
-        capacity to double the number of COVID patients hospitalized. Learn more
-        about{' '}
-        <ExternalLink href="https://docs.google.com/document/d/1cd_cEpNiIl1TzUJBvw9sHLbrbUZ2qCxgN32IqVLa3Do/edit">
-          our methodology
-        </ExternalLink>{' '}
-        and{' '}
-        <ExternalLink href="https://docs.google.com/presentation/d/1XmKCBWYZr9VQKFAdWh_D7pkpGGM_oR9cPjj-UrNdMJQ/edit">
-          our data sources
-        </ExternalLink>
-        .
-      </Fragment>
-    );
-  }
+  return (
+    <Fragment>
+      ICU data sourced from the{' '}
+      <ExternalLink href="https://healthdata.gov/dataset/covid-19-reported-patient-impact-and-hospital-capacity-state-timeseries">
+        Department of Health and Human Services (HHS)
+      </ExternalLink>
+      . As of December 21, we use "{ICUHeadroomMetric.extendedMetricName}"
+      instead of "ICU headroom used" as our primary ICU metric. Learn more about
+      our{' '}
+      <ExternalLink href="/covid-risk-levels-metrics#icu-capacity-used">
+        ICU Capacity methodology
```

Comment:
should capacity be capitalized? my first thought is it should be a lowercase c

Reply:
It was capitalized before, but I think you're right.  Fixed!